sklearn/tree/export.py

```diff
@@ -221,41 +222,64 @@ def node_to_str(tree, node_id, criterion):
         # Write node sample count
         if labels:
             node_string += 'samples = '
-        if proportion:
+        if proportion==True:
             percent = (100. * tree.n_node_samples[node_id] /
                        float(tree.n_node_samples[0]))
             node_string += (str(round(percent, 1)) + '%' +
                             characters[4])
+        elif proportion=='both':
+            node_string += (str(tree.n_node_samples[node_id]))
```

Comment:
Might be better to write `curr_n_node_samples = tree.n_node_samples[node_id]` somewhere at the top to make it easier to read the code